OpenAI-DotNet/BaseResponse.cs

```diff
@@ -22,5 +22,49 @@ public abstract class BaseResponse
         /// </summary>
         [JsonIgnore]
         public string RequestId { get; internal set; }
+
+        /// <summary>
+        /// The version of the API used to generate this response, as reported in the response headers.
+        /// </summary>
+        [JsonIgnore]
+        public string OpenAIVersion { get; internal set; }
+
+        /// <summary>
+        /// The maximum number of requests that are permitted before exhausting the rate limit.
+        /// </summary>
+
+        [JsonIgnore] 
+        public int LimitRequests { get; internal set; }
+
+        /// <summary>
+        /// The maximum number of tokens that are permitted before exhausting the rate limit.
+        /// </summary>
+        [JsonIgnore]
+        public int LimitTokens { get; internal set; }
+
+        /// <summary>
+        /// The remaining number of requests that are permitted before exhausting the rate limit.
+        /// </summary>
+        [JsonIgnore]
+        public int RemainingRequests { get; internal set; }
+
+        /// <summary>
+        /// The remaining number of tokens that are permitted before exhausting the rate limit.
+        /// </summary>
+        [JsonIgnore]
+        public int RemainingTokens { get; internal set; }
```

Comment:
lets make all these nullable 
```suggestion
        public int? RemainingTokens { get; internal set; }
```